SimPEG/electromagnetics/utils/AnalyticUtils.py

```diff
@@ -35,6 +36,12 @@ def MagneticDipoleVectorPotential(srcLoc, obsLoc, component, moment=1.,
     """
     # TODO: break this out!
 
+    warnings.warn(
```

Comment:
:wave: Any concerns (in particular, @sgkang) if we actually deprecate this code now rather than throwing the warning?  The mag dipole and loop code is now all in geoana